bookea-frontend/src/modules/user/pages/UserProfilePage.module.css

```
@@ -0,0 +1,97 @@
+    left: 0;
+    width: 100%;
+    height: 3px;
+    background: var(--bookea-orange);
+}
+
+/* Animations d'entrée */
+.fadeIn {
+    animation: fadeIn 0.4s ease-out;
+}
+
+@keyframes fadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)